index.html

```diff
@@ -5610,21 +5613,31 @@ <h3>Language and Base Direction</h3>
 
         <pre class="example nohighlight" title="Design pattern for natural language strings">
 "<a>property</a>": {
-  "value": "<span class="highlight">The string value</span>",
-  "lang": "<code>LANGUAGE</code>"
-  "dir": "<code>DIRECTION</code>"
+  "@value": "<span class="highlight">The string value</span>",
+  "@language": "<code>LANGUAGE</code>"
+  "@direction": "<code>DIRECTION</code>"
 }
         </pre>
 
+        <p>
+When the language value object is used in place of a string value, the object
+MUST contain a `@value` property whose value is a string, and SHOULD contain a
+`@language` property whose value is a string containing a well-formed
```

Comment:
Consider making "well-formed" be "valid" here. Conformance should only require well-formed, but authors should choose valid language tags. If it is too difficult to make the emit/consume distinction here, this text is fine.